### DISCUSSION TITLE
25/56 ... wording of the task

### DISCUSSION THREAD

**Discussion by dulhaver** [aykutkardas/regexlearn.com#254] [General]
First of all thx a mil for making this. However to me the wording of 25/56 is completely confusing to me

> Practice: Asterisk *
> 
> Use the asterisk * to write the expression that will select each word, suitable for the **absence of the letter e** in the text and the presence of one or more.

the solution `\de*p\g` actually shows **all occurences of the letter `e`** and not its absence.

would ...

> Use the asterisk * to write the expression that will select each word, suitable for the **presence** of the letter e plus the presence of one or more occurences in the text .

... be clearer ?

**Comment by aykutkardas** [comment]
I agree that this section should be more descriptive. I will consider your suggestion.
